news/4.34/platform.html

```diff
@@ -181,6 +181,39 @@ <h2>General Updates </h2>
         </p>
       </td>
     </tr>
+    
+    <tr id="edge-browser-improvements"> <!-- https://github.com/eclipse-platform/eclipse.platform.swt/issues/1466 -->
+      <td class="title">Improvements to Edge Browser (Windows only)</td>
+      <td class="content">
+        <p>
+          In an effort to make it the default browser for Windows, several issues that affected the Edge browser 
+          have been addressed in this release. Some of them are:
+          <ul>
+          	<li>Added support for basic authentication</li>
+          	<li>Inherited the proper light/dark theme from Eclipse instead of Windows</li>
+          	<li>Fixed obscure "No more handles" issues</li>
+          	<li>Fixed deadlocks that occurred due to the way Edge handles requests</li>
+          	<li>Fixed some styling issues in the welcome page for Eclipse</li>
+          	<li>Fixed some issues with javascript being used in an empty (local) page</li>
```

Comment:
Which kind of JavaScript script fix on "empty" pages does this refer to?
```suggestion
          	<li>Fixed some issues with JavaScript being used in an empty (local) page</li>
```

Reply:
Done